doc/ExpertOptions.md

```diff
@@ -1243,44 +1256,40 @@ By default, job results are [reused](https://documentation.dnanexus.com/user/run
   "ignoreReuse" : true
```

Comment:
in extras.json?

Reply:
It is part of "using the extras" section. But I added it to make the instruction clearer :)